mobile-app/lib/ui/views/learn/superblock/superblock_view.dart

```diff
@@ -3,6 +3,7 @@ import 'package:freecodecamp/models/learn/curriculum_model.dart';
 import 'package:freecodecamp/service/authentication/authentication_service.dart';
 import 'package:freecodecamp/ui/views/learn/block/block_view.dart';
 import 'package:freecodecamp/ui/views/learn/superblock/superblock_viewmodel.dart';
+import 'package:freecodecamp/ui/widgets/drawer_widget/drawer_widget_view.dart';
```

Comment:
```suggestion
```

Drawer is not needed in SuperBlock view and it prevents the user from going back to Learn Landing View without adding the Learn Landing View again into the navigation stack.